welcome-message/index.js

```diff
@@ -1,22 +1,103 @@
 const superstore = require('superstore-sync');
-
-import { $ } from '../utils';
+const expander = require('o-expander');
```

Comment:
I started with this and switched to o-expander, iirc for the event handlers and because toggler doesn't necessarily hide things